api/README.md

```diff
@@ -77,9 +80,21 @@ This API uses token-based authentication. A token is obtained by registering a n
 
 **Body:**
 ```json
-{
-    "email": "something@email.com",
-    "password": "123456"
+{	
+	"firstname":"Espoir",
+	"lastname":"Murhabazi",
+	"user_title":"Ir",
+    "email": "sampel@gmail.cim",
+    "nationality_country_id":"243",
+    "residence_country_id":"243",
+    "user_gender":"M",
+    "department":"Computer Science",
+    "user_disability":"none",
+    "user_category_id":1,
+    "affiliation":"Home",
+    "user_primaryLanguage":"english", 
+    "user_dateOfBirth":"2020-02-06T14:49:16.725Z",
+    "password": "***********"
```

Comment:
Currently this only requires the following fields -title, firstname, lastname, email address - please remove the rest.

Reply:
Thanks, @KaleabTessera for the feedback, The reason why I put the rest it's because when I was trying the endpoint it was. refusing and returning a 400 error without those fields!

Reply:
Cool, please let me know if this is still an issue. I will fix it if it is.